Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -42,18 +39,16 @@ enum BTPayPalError: Error, CustomNSError, LocalizedError {
             return 1
         case .fetchConfigurationFailed:
             return 2
-        case .httpResponseMissingUserInfoJSON:
-            return 3
         case .httpPostRequestError:
-            return 4
+            return 3
```

Comment:
This only is changing in the `convert-paypal` branch. So there are no breaking changes to call out in the CHANGELOG with this change in raw value of the errors. The new errors are all listed in the CHANGELOG for `convert-paypal`.